project/plugins.sbt

```diff
@@ -4,4 +4,6 @@ addSbtPlugin("org.scala-js"       % "sbt-scalajs"    % "0.6.22")
 addSbtPlugin("com.47deg"          % "sbt-microsites" % "0.7.18")
 addSbtPlugin("com.geirsson"       % "sbt-scalafmt"   % "1.5.1")
 addSbtPlugin("com.dwijnand"       % "sbt-dynver"     % "3.0.0")
+addSbtPlugin("org.scoverage"      % "sbt-scoverage"  % "1.5.1")
+addSbtPlugin("org.tpolecat"       % "tut-plugin"     % "0.6.3")
```

Comment:
how did we have `tut` before? does it come with `sbt-microsites`?

Reply:
Yes.